core/modules/server/server.js

```diff
@@ -226,6 +227,29 @@ Server.prototype.isAuthorized = function(authorizationType,username) {
 	return principals.indexOf("(anon)") !== -1 || (username && (principals.indexOf("(authenticated)") !== -1 || principals.indexOf(username) !== -1));
 }
 
+Server.prototype.isOriginWhitelisted = function(origin) {
```

Comment:
I try to avoid "whitelist" and "blacklist" in favour of "accept list" and "deny list" or "block list"

Reply:
Good tip, will do.